docs/guides/managed-backup/dedicated-backup-namespace-auto-backup/index.md

```diff
@@ -0,0 +1,407 @@
+---
+title: Define Cluster Wide Backup Policy using Stash Auto-backup
+description: A guide on how to define cluster wide backup policy using Stash auto-backup.
+menu:
+  docs_{{ .version }}:
+    identifier: managed-backup-dedicated-backup-namespace-auto-backup
+    name: Define Cluster Wide Backup Policy using Stash Auto-backup
+    parent: managed-backup
+    weight: 30
+product_name: stash
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+# Define Cluster Wide Backup Policy using Stash Auto-backup
+
+This guide will show you how you can define cluster wide backup policy using Stash Auto-backup.
+
```

Comment:
Here, we are going to define a backup policy for MySQL databases. Then, we will be demonstrate how users from different namespaces can use that policy to backup their own MySQL instances.